packages/firebase/index.d.ts

```diff
@@ -8303,31 +8319,71 @@ declare namespace firebase.firestore {
     INTERNAL: { delete: () => Promise<void> };
   }
 
+  /**
+   * Represents the task of loading a Firestore bundle. It provides progress of the bundle
+   * loading, task completion and error events should they be any.
+   *
+   * It can be used as a `Promise<LoadBundleTaskProgress>`.
+   */
   export interface LoadBundleTask {
+    /**
+     * Registers functions to listen to bundle loading progresses.
+     * @param next Called there is a progress update from bundle loading, typically whenever
+     *        a Firestore document is loaded it will generate a progress update.
+     * @param error Called when there is an error occurred from loading the bundle. The task
+     *        aborts after reporting the error, and there should be no more updates after this.
+     * @param complete Called when the loading task is complete.
+     */
     onProgress(
       next?: (progress: LoadBundleTaskProgress) => any,
       error?: (error: Error) => any,
       complete?: () => void
     ): void;
 
+    /**
+     * Implements a `Promise<LoadBundleTaskProgress>` interface.
+     *
+     * @param onFulfilled It is called with the compeltion `LoadBundleTaskProgress` when the
+     *        loading task completes.
+     * @param onRejected It is called when there is an error occurred from loading the bundle.
+     */
     then<T, R>(
       onFulfilled?: (a: LoadBundleTaskProgress) => T | PromiseLike<T>,
       onRejected?: (a: Error) => R | PromiseLike<R>
     ): Promise<T | R>;
 
+    /**
+     * Implements a `Promise<LoadBundleTaskProgress>` interface.
+     *
+     * @param onRejected It is called when there is an error occurred from loading the bundle.
+     */
     catch<R>(
       onRejected: (a: Error) => R | PromiseLike<R>
     ): Promise<R | LoadBundleTaskProgress>;
   }
 
+  /**
+   * Represents a progress update or a final state from loading bundles.
+   */
   export interface LoadBundleTaskProgress {
+    /** How many documents have been loaded. */
```

Comment:
I noticed that you removed these in the other PR. You should leave the comments in both places as firestore-exp builds its d.ts file from source.

Reply:
Yeah, that was a bad merge i think. Will keep the comments.